mos.yml

```diff
@@ -89,13 +89,15 @@ config_schema:
 
   - ["lb", "o", {title: "Light bulb settings", abstract: true}]
   - ["lb.name", "s", "Light bulb", {title: "Accessory name"}]
+  - ["lb.svc_type", "i", 0, {title: "HAP service type, -1 = disable (if available), 0 = normal"}]
```

Comment:
is there a reason this is not a simple boolean? will there be more service types?